src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.test.tsx

```diff
@@ -1,13 +1,22 @@
 import { render, screen } from 'test-utils';
+import { CityWeatherShortcutLoaderTestIds } from 'components/CityWeatherShortcut/components/CityWeatherShortcutLoader';
 import { NearestCitiesWeatherWidget } from './NearestCitiesWeatherWidget';
 import { NearestCitiesWeatherWidgetTestIds } from './NearestCitiesWeatherWidgetTestIds';
 
 describe('NearestCitiesWeatherWidget', () => {
-  it('should render', async () => {
+  it('should render', () => {
     render(<NearestCitiesWeatherWidget loading={false} cityName={undefined} />);
 
     const widget = screen.getByTestId(NearestCitiesWeatherWidgetTestIds.Widget);
 
     expect(widget).toBeInTheDocument();
   });
+
+  it('should loadrs', async () => {
```

Comment:
**typo** It could be `'should render loaders'` :)

Reply:
Right